install/apm/php/targeted-install/install.yml

```
@@ -1,4 +1,4 @@
-id: php-agent
+id: php-fpm-agent
```

```suggestion
id: php-targeted-install
```
Should probably update this id so it matches the directory